tools/cibadmin.c

```diff
@@ -388,6 +389,33 @@ static GOptionEntry addl_entries[] = {
       INDENT "Default value: 'auto'",
       "[value]" },
 
+    { "score-update", 0, G_OPTION_FLAG_NONE, G_OPTION_ARG_NONE,
```

Comment:
just "score" should be fine

Reply:
That is more concise and the description goes into detail, so I'm okay with it. One counterpoint: the value is not treated as a score in general. `attr="1000001"` will set `1000001`, not `1000000` (INFINITY).

Reply:
Yeah that is unfortunate but the description should be sufficient